src/FFmpeg.cpp

```diff
@@ -331,16 +336,35 @@ bool FindFFmpegLibs(wxWindow* parent)
 
    path = fd.GetLibPath();
 
+   const wxFileName fileName(path);
+
+   if (fileName.FileExists())
+      path = fileName.GetPath();
+
    wxLogMessage(wxT("User-specified path = '%s'"), path);
 
-   if (!::wxFileExists(path)) {
-      wxLogError(wxT("User-specified file does not exist. Failed to find FFmpeg libraries."));
+   wxString prevPath;
+   const bool hasUserPath = AVFormatPath.Read(&prevPath);
+
+   AVFormatPath.Write(path);
```

Comment:
To be very careful I might try-catch around what follows to guarantee that side effect on Preferences is undone, attempting a strong guarantee...

Or do it somehow with finally